README.md

```diff
@@ -23,11 +23,24 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+This library needs these to be in order to build and run:
+
+- Node 16 or Node 18
+- Typescript 4
+- Playwright (with a headless browser preferably chromium)
+- [Wrangler](https://developers.cloudflare.com/workers/wrangler/install-and-update/)
+
 ## Setup process
 The process consists of two steps. One needs to set up worker creation together with our support at [support@fingerprint.com](mailto:support@fingerprint.com). Afterwards, the Fingerprint Pro JS agent on the site needs to be configured to communicate with the worker.
 
 You can find the full documentation at [https://dev.fingerprint.com/docs/cloudflare-integration](https://dev.fingerprint.com/docs/cloudflare-integration).
 
+## Support
+
+To report problems, ask questions or provide feedback, please use [Issues](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/issues). If you need private support, you can email us at [oss-support@fingerprint.com](mailto:oss-support@fingerprint.com).
```

Comment:
Is `oss-support@fingerprint.com` the correct email address here?

Reply:
Yes, according the guideline "Writing DX GitHub Readmes" provided by Juraj at our Notion documents.